discovery/spec.md

```diff
@@ -1,51 +1,41 @@
-# CloudSubscriptions: Discovery - Version 0.1-wip
+# Discovery Service - Version 0.2-wip
 
 ## Abstract
 
-CloudSubscriptions Discovery API is a vendor-neutral API specification for
-determining what events are available from a particular service, as well as how
-to subscribe to those events.
+The Discovery Service specification allows for the discovery of messaging
+Endpoints and their related metadata such as Message Definitions and the
+mechanisms by which interactions with those Endpoints can be established.
 
 ## Table of Contents
 
 - [Overview](#overview)
 - [Notations and Terminology](#notations-and-terminology)
 - [Resource Model](#resource-model)
```

Comment:
I would list out the resources here in an expanded table to be able to find the specific resource in question.

Reply:
done